scripts/importer.py

```
@@ -0,0 +1,359 @@
+                    shop.get("phone"),
+                    shop["address"],
+                    True,
+                    shop.get("lat"),
+                    shop.get("lon"),
+                    city_id,
+                    now,
+                    None,
+                ),
+            )
+            imported += 1
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)